glide.yaml

```diff
@@ -1,31 +1,30 @@
 package: github.com/mysterium/node
+homepage: https://mysterium.network
+license: GPL-3.0
 import:
 - package: github.com/cihub/seelog
-  version: ^2.6.0
```

Comment:
It's common practice to allow build version change, i.e. `2.6.1` version should not change behaviour, but will probably contain some fixes.

Reply:
Also, that's why we have `glide.lock` to freeze build versions, but changing `glide.yaml` every time we want to get different build is an overkill and will likely lead to missing important fixes.

Reply:
My intention was to upgrade 1 package, but Glide does not have this feature. Just `glide update`
So decision is:
1. Developer will not randomly brake packages then: *adding/removing/updating 1 package* (was happening too frequently)
2. Then we want to *update all packages*, we: `update glide.yaml` -> runs `glide update`

@zolia @tadovas @donce